app/assets/javascripts/task/plugins.js.coffee

```diff
@@ -0,0 +1,7 @@
+@Plugins =
```

Comment:
I can't comment on the name, but I thought the preferred convention now was `file.coffee`, omitting the `.js`